plugin/core/documents.py

```diff
@@ -161,7 +161,7 @@ def notify_did_close(view: sublime.View):
 def notify_did_save(view: sublime.View):
     file_name = view.file_name()
     if file_name:
-        if file_name in document_states:
+        if has_document_state(view.window(), file_name):
```

Comment:
`view.window()` can return None - best to store in a `window` variable and check `if file_name and window:`

Reply:
Oh didn't know it can be None.. wonder what causes that. I'll change it